frontends/web/package.json

```diff
@@ -6,7 +6,7 @@
     "postinstall": "npx tcm src --camelCase dashes",
     "build": "preact build --no-prerender --service-worker=false",
     "dev": "preact watch",
-    "lint": "eslint --cache --ignore-path ../../.gitignore \"**/*.{js,jsx}\"",
+    "lint": "eslint --cache --ignore-path ../../.gitignore '**/*.{js,jsx}'; exit_code_eslint=$?; tslint -e 'src/**/*.css.d.ts' 'src/**/*.ts?(x)'; exit_code_tslint=$?; if [ $exit_code_eslint -gt 0 -o $exit_code_tslint -gt 0 ]; then exit 1; fi",
```

Comment:
Not sure `exit_code_xxx=$?` is going to work on Windows? Unless this is never run on Windows. Just checking.

Reply:
Also, I'd use `-ne` instead of `-gt` to be more future proof.